third_party/2/sqlalchemy/sql/sqltypes.pyi

```diff
@@ -23,7 +23,11 @@ class SmallInteger(Integer): ...
 class BigInteger(Integer): ...
 class Numeric(TypeEngine, _DateAffinity): ...
 class Float(Numeric): ...
-class DateTime(TypeEngine, _DateAffinity): ...
+class DateTime(TypeEngine, _DateAffinity):
+    def __init__(self, timezone=None): ...
+    @property
+    def python_type(self): ...
```

Comment:
Actually, instead of here, add this property on TypeEngine, it's defined there and available on all types.